### PR TITLE
(cleanup) - add node modules in gitignore

### PR DESCRIPTION
**Pull request by thisguychris**
some general cleanup:
- add node_modules in `.gitignore`
- rebuild the package-lock.json (was failing in node 10)
- remove unused variables in render for `base.js`

### CODE REVIEW

File: src/base.js [hunk 1]
```diff
@@ -92,17 +92,6 @@ export default class FormBase extends React.Component {
...
-      rules,
-      errorMessages,
-      attributeNames,
-      values,
-      onValues,
-      errors,
-      onErrors,
-      onValue,
-    } = this.props;
     return <React.Fragment>{this.renderChildren(children)}</React.Fragment>;
```
Comment (audiolion) [inline src/base.js]: `this.renderChildren(this.props.children)`?

File: package.json [hunk 2]
```diff
@@ -7,26 +7,37 @@
     "example": "example"
   },
   "jest": {
-    "testMatch": ["<rootDir>/tests/**/*.js"],
+    "testMatch": [
```
Comment (audiolion) [inline package.json]: this entire file is purely style changes, our prettier setup must be different from yours, we would prefer to keep it as is so could you remove this file from the commit?

Comment (thisguychris) [reply package.json]: okay will do

**Comment by thisguychris** [comment]
@audiolion the style changes in package.json was made by `npm`, but I did revert it back to original since its mostly cosmetic changes, updated the PR.

**Comment by zackify** [comment]
Hey... im sorry we forgot about this. If you want to update this to just change the base and gitignore, i will merge

**Pull request closed without merging**